kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java

```diff
@@ -1,4 +1,43 @@
 package com.kodilla.testing.collection;
 
+import org.junit.jupiter.api.*;
+
+import java.util.Arrays;
+import java.util.List;
+
 public class CollectionTestSuite {
+    @BeforeEach
+    public void before() {
+        System.out.println("Test Case: begin");
+    }
+
+    @AfterEach
+    public void after() {
+        System.out.println("Test Case: end");
+    }
+
+    @DisplayName("When create list of numbers, then oddNumberExterminator should return only even numbers")
+    @Test
+    void testOddNumberExterminatorNormalList() {
+        //Given
+        OddNumberExterminator oddNumberExterminator = new OddNumberExterminator();
```

Comment:
oddNumberExterminator jest bezstanowy, wiec moze byc zmienna klasowa. 
`private OddNumberExterminator ...`

Reply:
Zrobiłem oddNumberExterminator jako private ale teraz nie mogę uruchomić testu? Czy powinnienem tą funkcję wywołać do przeprowadzenia testu?

Reply:
Mysle ze sie nie zrozumielismy. W klasie CollectionTestSuite dodaj 
```
private OddNumberExterminator oddNumberExterminator = new OddNumberExterminator();
```
+ usun z lini 23 i 35 exterminator

Reply:
Chyba ze jest inny powod dla ktorego nie mozesz uruchomic testu

Reply:
![Bez tytułu](https://user-images.githubusercontent.com/110178231/184334839-87a5ad67-ac19-4f84-ae00-225d939c3b36.png)
bez private wszystko działa, jak dodam private to program sie nie kompiluje